health-monitors/gpu-health-monitor/Makefile

```diff
@@ -204,6 +204,44 @@ format:
 	@echo "Running Black formatter on $(MODULE_NAME)..."
 	poetry run black .
 
+format-protobuf-files:
+	@echo "Formatting local protobuf files for $(MODULE_NAME)..."
```

Comment:
I'm not sure if we want this, we should keep the proto files as is when they are generated. What do you think?